### PR TITLE
Source Salesloft: update expected records

### PR DESCRIPTION
**Pull request by artem1205**
## What

Fix Source Salesloft Connector health report

## How
Fix expected records

## Recommended reading order
1. `expected_records.jsonl`

## 🚨 User Impact 🚨

no breaking changes

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-salesloft test report (commit [`4c833ffe1f`](https://github.com/airbytehq/airbyte/pull/27484/commits/4c833ffe1ff66bddf8f3fd43eb8ebccd943f8405)) - ❌

⏲️  Total pipeline duration: 318 seconds

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-salesloft/metadata.yaml | ✅        |
| Connector version semver check.                                         | ✅        |
| Connector version increment check.                                      | ❌        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-salesloft docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5315447023)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-salesloft test
```

**Comment by octavia-squidington-iii** [comment]
##  source-salesloft test report (commit [`4c833ffe1f`](https://github.com/airbytehq/airbyte/pull/27484/commits/4c833ffe1ff66bddf8f3fd43eb8ebccd943f8405)) - ❌

⏲️  Total pipeline duration: 301 seconds

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-salesloft/metadata.yaml | ✅        |
| Connector version semver check.                                         | ✅        |
| Connector version increment check.                                      | ❌        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-salesloft docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5315448789)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-salesloft test
```

File: airbyte-integrations/connectors/source-salesloft/integration_tests/expected_records.jsonl [hunk 1]
```diff
@@ -44,17 +44,17 @@
...
+{"stream":"actions","data":{"id":343305169,"due":true,"created_at":"2023-03-07T12:12:53.201684-05:00","updated_at":"2023-06-17T09:53:56.994997-04:00","type":"email","status":"in_progress","due_on":"2023-03-07T00:00:00.000000+00:00","multitouch_group_id":null,"action_details":{"_href":"https://api.salesloft.com/v2/action_details/email_details/198394","id":198394},"user":{"_href":"https://api.salesloft.com/v2/users/6970","id":6970},"person":{"_href":"https://api.salesloft.com/v2/people/103358221","id":103358221},"cadence":{"_href":"https://api.salesloft.com/v2/cadences/25591","id":25591},"step":{"_href":"https://api.salesloft.com/v2/steps/198394","id":198394}},"emitted_at":1687198842250}
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-salesloft/integration_tests/expected_records.jsonl:47]: I suggest you add the `updated_at` field to ignored fields in the CAT config

**Review by davydov-d — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-salesloft test report (commit [`f51cbee6a4`](https://github.com/airbytehq/airbyte/pull/27484/commits/f51cbee6a4bcd57d4f34a19f3dacea91a66b709c)) - ❌

⏲️  Total pipeline duration: 313 seconds

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-salesloft/metadata.yaml | ✅        |
| Connector version semver check.                                         | ✅        |
| Connector version increment check.                                      | ❌        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-salesloft docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5316071475)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-salesloft test
```

**Comment by octavia-squidington-iii** [comment]
##  source-salesloft test report (commit [`6b380464dd`](https://github.com/airbytehq/airbyte/pull/27484/commits/6b380464dd4bc1903b12fd8c74389d867310a87f)) - ✅

⏲️  Total pipeline duration: 313 seconds

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-salesloft/metadata.yaml | ✅        |
| Connector version semver check.                                         | ✅        |
| Connector version increment check.                                      | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-salesloft docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                              | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5316153042)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-salesloft test
```

**Pull request merged**